GalenAnswers.md

```diff
@@ -0,0 +1,90 @@
+# Answers
+
+## HPC good citizenship
+
+1. Using "-pe openmp 64" does not mean that commands will use multiple processors, just that 64 cores will be reserved for your job. As far as I can tell, there's nothing to stop someone from submitting an extremely simple script with a single command and requesting 64 cores for it, but the command would still only run on one core.
+
+2. One can find this information by running a smaller test dataset prefixed by the "/usr/bin/time -v" command. For example, running the commands 
+```
+touch timetest.txt
+/usr/bin/time -v echo "This is me testing the time command" >> timetest.txt
```

Comment:
this is good but it is is also good to add option -o which allows an output file. env time -v -o MYOUTPUT CMD would do the trick and do a dump to a text file. this helps sometimes if you care about what goes in your stdout and stderr files. I like to keep it seperate as I usually do tail -n 1 MYSTDOUTFILE to see if I was able to run to completion and then sort my STDERRFILES using ls -lS jobname.ejobid* by size to see if the stderr produced larger output than normal. this typically means there where errors on the run. This doesn't work though when the programmers are sloppy and lazy and output stdout to stderr. :(